src/validation.h

```diff
@@ -935,6 +936,11 @@ class ChainstateManager
         return m_blockman.m_block_index;
     }
 
+    /**
+     * Track versionbit status
+     */
+    mutable VersionBitsCache m_versionbitscache;
```

Comment:
Could you explain a bit why `m_versionbitscache` is marked `mutable`?

Reply:
It's a cache, so its existence doesn't impact the logical API of the class -- if we didn't use a cache, you could have a `const ChainstateManager` and get answers as to whether a deployment is active/enabled and what version to use for new blocks; so adding a cache shouldn't mean that you can only do that with a non-const `ChainstateManager`.

ie, I'm treating `const ChainstateManager` as being "logically" constant, not bitwise constant.

Reply:
Sounds good!